### ISSUE TITLE
(French) accents and &

### ISSUE THREAD

**Issue by sJARB** [jhhoward/MicroWeb#12]
Hi all, I discover this incredible project by the webmaster of a famous French retro informatic forum, Win3x.org. 
I've installed MicroWeb under PC-DOS V/7.0 (Japanese version of PC-DOS 2000) to have all DOS Web Browsers in the same VM (under 86box, Pentium MMX/166, 32 MB RAM, CL-GD 5480, PC-Net ISA 0x300 IRQ3), and I report some issues I've encounter.

1° HTML entities like &#224; (à) or others, can't display. I'm sad to not see French accents on my website http://winjoy.sjarb.fr (the same for Greek letters to a lesser extent)

2° I think, <table> is not supported yet, because the one I use for the header seems not work here. The cells are displayed one below the other

3° When I try to use JavaScript, MicroWeb don't display anything

4° WINJOY is a verifed XHTML 4.0 Transitional site, but to do that, I'm foced to use an HTML entity for & instead, because I'm using parameters in URLs, but, as MicroWeb don't support these, it tries to access to the page without replace the entity by a real &. So I've to manually replace it in the search bar

That's all for today, thanks for your good work (^_^)

![20211112-080257-683](https://user-images.githubusercontent.com/75838352/141432166-bece2a0e-f739-4683-80cf-354b7c44ed20.png)
Home page of WINJOY (No French accents, no Greek letters, no table)

![20211112-081322-064](https://user-images.githubusercontent.com/75838352/141433456-a4d57ff0-3a63-4d35-9702-f03cafab6e73.png)
When i'm trying to access WINJOY with JavaScript

![20211112-080307-230](https://user-images.githubusercontent.com/75838352/141432170-65b921fa-cb63-4d97-9097-01933c071263.png)
When I'm trying to acces to the Windows section

**Comment by jhhoward** [comment]
Thanks for reporting on how your site performs! Currently accented characters get replaced with their not-accented versions. I'm planning to support a wider variety of characters in the future.

**Comment by sJARB** [comment]
The major issue is just html entities. I'm using that because of poor support in IE3. But MicroWeb don't pay attention to that. I'm making an other version with only the accented characters, and where I convert these to entites if necessary

**Comment by jhhoward** [comment]
Version 2.0 now supports tables and a subset of accented characters. The URL you shared is no longer working so unfortunately I couldn't test with your site. Let me know if it displays correctly now :)

**Comment by sJARB** [comment]
Yes it's true, I moved my site into http://sjarb.fr since my last post 😅 it's midnight here in France, I have no time to test now

**Comment by sJARB** [comment]
I've test the "new" microweb 2.0, and everything is beautiful now 🤩🤩
Colors are well displayed (with a good palette), decimal entites also (for instance, "&#233;" is correctly displayed "é" but not "&eacute;", and it now ignore all script tags.

![Monitor_1_20240830-162552-826](https://github.com/user-attachments/assets/f981092b-a0d4-46c5-9868-19338243b1ff)
Monochrome VGA

![Monitor_1_20240830-162516-182](https://github.com/user-attachments/assets/e69ea249-52fa-4a9a-ab06-26635c39cea7)
16-color VGA (i don't have an optimized palette for 16-colors modes)

![Monitor_1_20240830-162346-086](https://github.com/user-attachments/assets/9b9ad5c6-0fca-4d46-b479-89a410193bd8)
256-color VGA (my palette is not fulle optimized for 256-colors modes 😅😅 unless Microweb dont support attributes "bgcolor" for "table" or "td" tags ???)
